python/doc/examples/data_analysis/manage_data_and_samples/plot_sample_manipulation.py

```diff
@@ -27,10 +27,6 @@
 # Get the mean per component
 sample.computeMean()
 
-# %%
-# Get the standard deviation per component
-sample.computeStandardDeviationPerComponent()
```

Comment:
I  think instead of deleting this, you could change it to
`sample.computeStandardDeviation()`

Of course, this means that the identical paragraph at lines 54-56 must be deleted:
```
# %%
# Get the sample standard deviation
sample.computeStandardDeviation()
```

Reply:
the result is the same isnt it ?

Reply:
Yes, but is seems more natural to have it at this spot in the example rather than at the bottom of the file. By deleting this paragraph, you are moving the "easy" standard deviation computation among more complicated methods, like the computation of the covariance matrix. What do you think @jschueller?

Reply:
ok